SD-K9/src/app/models/transport.enum.model.ts

```diff
@@ -2,5 +2,6 @@ export enum Transport {
     BICYCLING = 'BICYCLING',
```

Comment:
Is the bicycle option still relevant?

Reply:
yes, since some user ride their bikes to campus